site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -0,0 +1,371 @@
+        not the original due date.*/
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable',gradeable,'update?nav_tab=5']);
+        cy.get('#date_due')
+            .clear()
+            .type(getCurrentTime('threeDaysAgo'))
+            .type('{enter}');
+        cy.get('#save_status', {timeout:20000}).should('have.text', 'All Changes Saved');
+        cy.logout();
+        SubmitAndCheckMessage('non_team','upload_file2','invalid_4_days_late');
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable',gradeable,'update?nav_tab=5']);
+        cy.get('#no_late_submission').click(); //disable late submissions
+        cy.get('#save_status', {timeout:20000}).should('have.text', 'All Changes Saved');
+    });
+});
+
+describe('Test warning messages for team gradeable', () => {
+
+    it('should create team gradeable for testing', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable']);
+        // Enter gradeable info
+        cy.get('#g_title').type(team_gradeable);
+        cy.get('#g_id').type(team_gradeable);
+        cy.get('#radio_ef_student_upload').check().click();
+        cy.get('#team_yes_radio',{timeout:20000}).check().click();
+        // Create Gradeable
+        cy.get('#create-gradeable-btn').click();
+
+        //Date page
+        cy.get('#page_5_nav').click();
+        cy.get('#date_ta_view')
+            .clear()
+            .type('1992-06-15')
+            .type('{enter}');
+        cy.get('#save_status', {timeout:20000}).should('have.text', 'All Changes Saved');
+
+        cy.get('#date_submit')
+            .clear()
+            .type('2004-12-18')
+            .type('{enter}');
+        cy.get('#save_status', {timeout:20000}).should('have.text', 'All Changes Saved');
+        cy.get('#late_days')
+            .clear()
+            .type(3)
+            .type('{enter}');
+        cy.get('#save_status', {timeout:20000}).should('have.text', 'All Changes Saved');
+        //Create team
+        cy.logout();
+        cy.login('student');
+        cy.visit(['sample', 'gradeable',team_gradeable,'team']);
+        cy.get('#create_new_team').click();
+        cy.get('#invite_id').type('aphacker').type('{enter}');
+        cy.logout();
+        cy.login('aphacker');
+        cy.visit(['sample', 'gradeable',team_gradeable,'team']);
+        cy.get('#accept_invitation').click();
+    });
+
+    it('should show a confirmation message for the first submission', () => {
+        //Part 1/2 of a test case
+        //The first submission will be done 2 days after the due date and use 2 valid late days for each team member
+        cy.login('instructor');
+        giveExtentions(team_gradeable);
+        giveLateDays(getCurrentTime('threeDaysAgo'),'student',3); //this is important for part 2/2
+        giveLateDays(getCurrentTime('threeDaysAgo'),'aphacker',2);
+        cy.visit(['sample', 'gradeable',team_gradeable,'update?nav_tab=5']);
```

Comment:
I'm not really familiar with the linting style that Submitty uses, is there a spacing convention we use with lists? like no space after a comma, or a space after a comma? (Or does it not matter)

Reply:
We typically out a space after each comma in a list, but that isn't being enforced by our linter here.  This is a good chance to update our JS linting rules to standardize this!

Reply:
I added spaces after each comma in all lists